### ISSUE TITLE
DDBEnricherMixin.Z.generateTokenMagicFXChange is not a function

### ISSUE THREAD

**Issue by boyobejamin** [MrPrimate/ddb-importer#518]
**Describe the bug**

Can't import a character sheet created in DnDBeyond with the following error.

`Error Processing Character: TypeError: DDBEnricherMixin.Z.generateTokenMagicFXChange is not a function`

**To Reproduce**
1. Create a new Player Character or existing Player Character.
2. URL is https://www.dndbeyond.com/characters/126300002
3. Start Import (with all default settings left)

**Expected behavior**
Expected character to import without issue

**Screenshots**
![image](https://github.com/user-attachments/assets/69ad5eb5-39d4-44c0-9404-97225d69ef2d)

**Environment:**
 - Chrome
 - Foundry Version: 12 Stable Build 331
 - 5e System Version: 4.0.4
 - Module Version: 6.0.28

**Additional context**

 - [A Character URL, if the problem is a character import.](https://www.dndbeyond.com/characters/126300002)

```
logger.js:84 DDB Importer | ERR > TypeError: DDBEnricherMixin.Z.generateTokenMagicFXChange is not a function
logger.js:84 DDB Importer | ERR > TypeError: DDBEnricherMixin.Z.generateTokenMagicFXChange is not a function
DDB Importer | ERR > JSON Fetch and Parse Error
```

**Comment by boyobejamin** [comment]
This is a potential guess, our player recently added Starry Wisp which may have something to do with this? Just a guess!

https://www.dndbeyond.com/spells/2618840-starry-wisp

**Comment by MrPrimate** [comment]
Fixed in 6.0.29
